openshift/conf.sh

```diff
@@ -11,6 +11,8 @@ export THOTH_LOCAL=1
 
 # Adjust if you deploy to actual cluster - this is a namespace to separate different deployments:
 export THOTH_DEPLOYMENT_NAME="local-$USER"
+export THOTH_MIDDLEEND_NAMESPACE="thoth-test-dev"
+export THOTH_BACKEND_NAMESPACE="thoth-test-dev"
```

Comment:
If we want to move this to conf.sh, maybe it will be a good idea to place all namespace configuration here - there is missing THOHT_FRONTED_NAMESPACE.

Reply:
Check out the ansible playbooks, their functionality should be equivalent to conf/deploy and they work with and OpenShift Cluster, just provide OCP_TOKEN and OCP_URL